### ISSUE TITLE
Game softlocked when kerbal on EVA dies

### ISSUE THREAD

**Issue by Mathmagician8191** [UmbraSpaceIndustries/USI-LS#307]
I tested the kerbal EVA time expiry by using "set position" in the cheats menu to put a non-veteran kerbal on the surface of Duna.
I had the kerbal focused and was not timewarping when the kerbal died.
When the kerbal died, the game softlocked
Effects of the softlock:
* the pause menu was not appearing when I pressed Escape but the timer turned gray
* Time no longer advanced (a 10 second raw time alarm was stuck on 9 seconds, even while the timer was green)
* on the map view, Ike was in the center, and there was an orbit line with no object on it
When I closed the game using Alt+F4 and restarted it, things were back to normal and the save was a few minutes before the kerbal died

Version: 1.11.2.3077 (WindowsPlayer x64) en-us on Proton 6.3-4 with breaking ground

Mods:
USI Life support
Kerbal Alarm Clock
RCS Build Aid
Custom Asteroids
Community Tech Tree
Hide Empty Tech Tree Nodes
Some custom module manager patches (you can find them at https://github.com/Mathmagician8191/KSP-tweaks)

The module manager patch I used to adjust life support:
```
@LIFE_SUPPORT_SETTINGS:AFTER[UmbraSpaceIndustries]
{
  @SupplyTime = 21600
  @ECTime = 3600
  @NoSupplyEffect = 5
  @NoSupplyEffectVets = 5
  @NoECEffect = 5
  @NoECEffectVets = 5
  @EVAEffect = 5
  @EVAEffectVets = 5
  @NoHomeEffect = 1
  @NoHomeEffectVets = 1
  @HomeWorldAltitude = 6000
}
```

The reason I tested this was because the EVA time was only showing 1 hour in the life support window, but after each hour the timer refreshed until the 6 hour mark when they died (may have something to do with the ECTime?)

**Comment by arbsoup** [comment]
> The reason I tested this was because the EVA time was only showing 1 hour in the life support window, but after each hour the timer refreshed until the 6 hour mark when they died (may have something to do with the ECTime?)

No, it's because the EVA timer doesn't actually display hours for whatever reason:

https://github.com/UmbraSpaceIndustries/USI-LS/blob/c842b8587e150875cc563c3aca541d618d5aadbb/Source/USILifeSupport/LifeSupportMonitor.cs#L121

That's fixed in my PR too, it should probably just be `, timeString;`.
